src/main/java/hw7/entities/MetalsAndColorsData.java

```diff
@@ -0,0 +1,20 @@
+package hw7.entities;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.util.List;
+
+@AllArgsConstructor
+@Getter
+public class MetalsAndColorsData {
+    private List<String> summary;
+    private List<String> elements;
+    private String color;
+    private String metals;
+    private List<String> vegetables;
+
+    public MetalsAndColorsData(){
```

Comment:
What is the reason of NoArgsConstuctor here ?

Reply:
Jackson recuires it. Without default constructor it throws InvalidDefinitionException